data/mods/blazemod/vehicleparts/blob_engine.json

```diff
@@ -0,0 +1,25 @@
+[
```

Comment:
Separated out for now, I'll have to review how this part acts in-game to make sure it's working as intended.